src/app/_action.ts

```diff
@@ -0,0 +1,49 @@
+'use server';
+
+import { revalidatePath } from 'next/cache';
+import { createLoan, deleteLoan, updateLoan } from '@/lib/loan-db';
+import { TLoan } from '@/models/loan';
+
+/**
+ * Server Action: Create a new loan.
+ */
+export async function createLoanAction({
+	path,
+	params,
+}: {
+	path: string;
+	params: TLoan;
+}) {
+	await createLoan({ ...params });
+	revalidatePath(path);
```

Comment:
Question,w hat is revalidate Path doing?

Why we need the layer action? Can not we call directly to the `loan-db` from the frontend?

Reply:
When i will switch on next module i will fix this